### PR TITLE
chore: set up pull request template

### LINKED ISSUE #3: Set up organization wide pull request template
Ref: https://github.blog/changelog/2019-02-21-organization-wide-community-health-files/

### PR DESCRIPTION
**Pull request by afuetterer**
Fixes #3

I copied the version from the `dev-2.2.0` branch of the main rdmo repo:
https://github.com/rdmorganiser/rdmo/blob/dev-2.2.0/.github/PULL_REQUEST_TEMPLATE.md

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+<!--- If suggesting a new feature or change, please discuss it in an issue first. --->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce. --->
+
+<!--- Please provide a general summary of your changes in the title of the pull request --->
+
+## Description
+<!--- Describe your changes in detail --->
+
+<!--- Please link to a related issue here --->
+Related issue: #ISSUE_NUMBER
```
Comment (afuetterer) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should we prefer

`Fixes #ISSUE_NUMBER` or `Closes #ISSUE_NUMBER`?

This way GitHub will automatically link PR and issue (on the right side of the UI under "development"), meaning merging the PR will close the issue.

Whereas using `Related issue: #ISSUE_NUMBER` simply links them, merging does not close the issue.

Comment (jochenklar) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would prefer "Resolves: #issue" but it only works on the default branch: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Comment (afuetterer) [reply .github/PULL_REQUEST_TEMPLATE.md]: Alright. I think it needs to be `Resolves #ISSUE_NUMBER` (no colon). And yes, once the changes reach the default branch, issues will be closed.
Changing to `Resolves`.

**Comment by afuetterer** [comment]
How about this one?

**Pull request closed without merging**